extensions/wikia/InsightsV2/InsightsHooks.class.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+class InsightsHooks {
+
+	/**
+	 * Check if article is in insights flow and init script to show banner with message and next steps
+	 */
+	public static function onBeforePageDisplay( OutputPage &$out, Skin &$skin ) {
```

Comment:
No need for the `&` as objects are effectively passed by reference in PHP anyway.